firmware/application/apps/ui_adsb_rx.hpp

```diff
@@ -64,8 +64,8 @@ namespace ui {
 #define VEL_AIR_SUBSONIC 3
 #define VEL_AIR_SUPERSONIC 4
 
-#define O_E_FRAME_TIMEOUT 20     // timeout between odd and even frames
-#define MARKER_UPDATE_SECONDS 5  // "other" map marker redraw interval
+#define O_E_FRAME_TIMEOUT 20      // timeout between odd and even frames
+#define MARKER_UPDATE_SECONDS 10  // "other" map marker redraw interval
```

Comment:
Couldn't we keep 5 for the classic map mode ?
Like having:
#define CLASSIC_MAP_MARKER_UPDATE_SECONDS 5
#define OSM_MAP_MARKER_UPDATE_SECONDS 10
And initialize an uint8_t marker_update_seconds with the right value at class init, whe you are detecting if OSM have to be used instead of classic ?

Reply:
Nice idea! Will do

Reply:
Done